### PR TITLE
CVE release 1.10

### PR DESCRIPTION
**Pull request by anushkamittal2001**
Has new images with CVE fixes updated.

### CODE REVIEW

File: charts/enterprise-kyverno-operator/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: nirmata-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
 version: v0.3.5
-appVersion: v0.2.0
+appVersion: v0.1.19
```
Comment (pns-nirmata) [inline charts/enterprise-kyverno-operator/Chart.yaml:6]: Should be 0.2.1

File: charts/nirmata/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 name: kyverno
-version: 3.0.5-rc7
-appVersion: v1.10.4-n4k.nirmata.1
+version: 3.0.5-rc8
```
Comment (pns-nirmata) [inline charts/nirmata/Chart.yaml:4]: Let us make this 3.0.7, matching upstream

File: charts/enterprise-kyverno-operator/Chart.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ name: nirmata-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
 version: v0.3.5
```
Comment (pns-nirmata) [inline charts/enterprise-kyverno-operator/Chart.yaml:5]: 0.3.6

File: charts/enterprise-kyverno-operator/Chart.yaml (hunk 1 above)
Comment (anushkamittal2001) [reply charts/enterprise-kyverno-operator/Chart.yaml:6]: @pns-nirmata keeping this unchanged as discussed over call, unless you recommend otherwise.

Comment (pns-nirmata) [reply charts/enterprise-kyverno-operator/Chart.yaml:6]: Correct. Should be unchanged.

File: charts/enterprise-kyverno-operator/Chart.yaml [hunk 4]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: nirmata-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.3.5
-appVersion: v0.2.0
+version: v0.3.6
+appVersion: v0.1.19
```
Comment (pns-nirmata) [inline charts/enterprise-kyverno-operator/Chart.yaml:6]: Set this to v0.2.1.

**Review by pns-nirmata — APPROVED** [review]

**Pull request merged**